locations/spiders/pollotropical.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import re
+from locations.items import GeojsonPointItem
+
+class QuickleesSpider(scrapy.Spider):
+    name = "pollotropical"
+    allowed_domains = ["https://www.pollotropical.com/"]
+    item_attributes = {
+        "brand": "Pollo Tropical",
+        "brand_wikidata": "Q3395356",
+        "country": "USA",
+    }
+
+    start_urls = (
+        'https://www.pollotropical.com/locations',
+    )
+
+    def parse(self, response):
+
+        store = response.xpath(
+            "//div[@class='styles__StyledCardInfo-s1y7dfjk-3 fncRsG']/div/h3/a/text()").extract()
```

Comment:
This look really brittle. If I look at the web page you attempt to scrape there is a big JSON array:
**<script>__NEXT_DATA__ = {"props":{"pageProps":{"isServer":true,"store .......**
that you could load and walk through. The array has lat/lon points also.
Use dict_parser.DictParser() ... look for examples of use in the project ... to help you parse each store JSON?